sample-data/postgres-insert-helpers.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+
+const {
+  generateListing,
+  generateReservation,
+  generateDailyPrices,
+} = require('./generator');
+
+// adds listings to csv
+
+fs.writeFileSync('./listings.csv', '');
+
+for (let i = 0; i < 100; i += 1) {
+  let listings = [];
+  for (let j = 0; j < 100000; j += 1) {
+    const listing = generateListing();
```

Comment:
great job using separation of concerns with your help functions!

Reply:
Thanks